cassandra_test.go

```diff
@@ -1977,6 +1977,40 @@ func TestNegativeStream(t *testing.T) {
 	}
 }
 
+func TestProtocolV2StreamExhaustion(t *testing.T) {
```

Comment:
A simpler test would be to use the unit test functions and just exhaust a connections streams.